src/utils/config.js

```diff
@@ -7,3 +7,20 @@ export function getPurposes(config){
     }
     return Array.from(purposes)
 }
+
+export function locationIsPrivacyPage(config){
+    const getPolicyUrl = () => {
+        try{
+            // Absolute path
+            return new URL(config.privacyPolicy).pathname
```

Comment:
Didn't use `.href` here because of a potential `#` after the pathname, for example set by the link to open the manager. This makes it somewhat imprecise though. We could strip `#` from the URL string alternatively.